### PR TITLE
172 automation - roxygen check - fail build on warning

### PR DESCRIPTION
**Pull request by tomszosz**
# Pull Request

Fixes https://github.com/insightsengineering/automation-tasks/issues/172

test action details: 
https://github.com/insightsengineering/test.nest/actions/runs/1563773859

### CODE REVIEW

**Comment by tomszosz** [comment]
result details:
image

**Review by dinakar29 — APPROVED** [review]
LGTM

File: .github/workflows/roxygen.yaml [hunk 1]
```diff
@@ -53,7 +53,26 @@ jobs:
...
+          logfile <- "roxygen_${{ github.event.repository.name }}.log"
+          con <- file(logfile)
+          sink(con, append = TRUE, split = TRUE)
+          sink(con, append = TRUE, type = "message")
           roxygen2::roxygenize('.', roclets = c('rd', 'collate', 'namespace'))
+          sink()
+          sink(type = "message")
+          logs <- readLines(logfile)
+          error_marker <- grep("Error:", logs)
+          warnings_marker <- grep("Warning message", logs)
```
Comment (arkadiuszbeer) [inline .github/workflows/roxygen.yaml:65]:
```suggestion
          warnings_marker <- grep("Warning message:", logs)
```
Maybe with `:`

**Review by arkadiuszbeer — APPROVED** [review]
LGTM

**Pull request merged**